setup.py

```diff
@@ -69,6 +90,7 @@ def get_extensions():
     description="Detectron2 is FAIR's next-generation research "
     "platform for object detection and segmentation.",
     packages=find_packages(exclude=("configs", "tests")),
+    data_files=get_model_zoo_configs(exclude=["quick_schedules", "Detectron1-Comparisons"]),
```

Comment:
I tried `package_data={"detectron2": ["checkpoint/configs/*.yaml", "checkpoint/configs/**/*.yaml"]},` and it works as expected, installing the config files to `detectron2/checkpoint/configs` after installation. It's much easier since the directory structure inside `checkpoint/configs/` will be kept automatically.

Reply:
Right, this hack was also from a previous trial where configs were still out of the package after build. Will fix.

Reply:
Did you mean to suggest using `package_data` instead of `data_files` or this exact snippet?

Because I haven't considered Detectron1-comparisons and quick_schedules as a part of Model Zoo.

Reply:
Yes. Having the extra configs doesn't hurt. In fact, our internal end-to-end tests can benefit from having quick_schedules as part of the package.